tests/test_shell_script_generator.py

```diff
@@ -122,7 +122,8 @@ def test_generate_shell_script(self):
         'project_id': 'my-project',
         'storage_location': 'us-east1',
         'shutdown_timer_in_sec': 500,
-        'base_image_family': 'projects/my-dataproc-project/global/images/family/debian-10'
+        'base_image_family': 'projects/my-dataproc-project/global/images/family/debian-10',
```

Comment:
please let's d use 12 if we can here.  10 will be slipping into the bit bucket with the impending release of 13, trixie, which is already in [testing](https://wiki.debian.org/DebianTrixie).